src/consensus/malachite/host.rs

```diff
@@ -197,7 +194,6 @@ impl Host {
                     }
                 };
                 let proposed_value = state.shard_validator.add_proposed_value(&proposal);
-                state.shard_validator.decide(commits).await;
```

Comment:
We should also add error or panic within `Proposer.decide` if we can't find the proposal to commit